src/containers/app/CreateApp.js

```diff
@@ -76,13 +80,13 @@ class CreateApp extends React.Component {
       .map(id => id.trim())
       .filter(id => !!id);
 
-    const app = {
-      description,
-      name,
-      title,
-      url,
-      appTypeIds: splitAppTypeIds
-    };
+    const app = (new AppBuilder())
```

Comment:
With this in place, you can now remove the `if (!title || !description || !name || !appTypeIds || !url)` from above, and do something like this:

```
try {
  const splitAppTypeIds = appTypeIds
  ...
  const app = (new AppBuilder())
  ...
  this.props.actions.createAppRequest(app);
}
catch (e) {
  this.setState({ isError: true });
}
```